lib/code_corps/policy/conversation.ex

```diff
@@ -32,4 +32,19 @@ defmodule CodeCorps.Policy.Conversation do
     conversation |> get_message() |> get_project() |> administered_by?(user)
   end
   def show?(_, _), do: false
+
+  def update?(%User{admin: true}, _conversation), do: true
+  def update?(%User{id: id} = user, %Conversation{} = conversation) do
+    # scoped_message_ids =
+    #   Message
+    #   |> Policy.Message.scope(user)
+    #   |> select([m], m.id)
+    #   |> Repo.all
+
+    # conversation
+    # |> where(user_id: ^id)
+    # |> or_where([c], c.message_id in ^scoped_message_ids)
+    true
```

Comment:
@joshsmith what are the rules here when updating a conversations?

Reply:
Good question. I think it should be updated really only by someone who's an admin or the message's project is administered by a user.

The `read_at` in a conversation is going to be set by an update to the conversation_part, so it won't happen explicitly. The only explicit update to a conversation _should_ be the `status`, which should be admin-only.

Reply:
lmk what you think!  updated everything.